testing/tests/DevExpress.exporter/imageCreator.tests.js

```diff
@@ -2652,3 +2557,66 @@ QUnit.test('Some items is async', function(assert) {
         done();
     });
 });
+
+QUnit.module('Tests with private API usage', {
```

Comment:
Please, use including modules (when we write tests exactly in module's body). See https://github.com/DevExpress/DevExtreme/blob/23_1/testing/tests/DevExpress.ui.widgets/popup.tests.js#L598